src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.ps1

```diff
@@ -19,7 +19,7 @@ Tests retrieval of classic administrators
 function Test-RaClassicAdmins
 {
     # Setup
-    $subscription = Get-AzureRmSubscription
+    $subscription = Get-AzureRmSubscription | Where-Object { $_.Id -ieq "4004a9fd-d58e-48dc-aeb2-4a4aec58606f" }
```

Comment:
@sneivandt do we need to include these specific subscriptions and accounts in all of the tests below? This makes it difficult for users who don't have access to these subscriptions and accounts to re-record these tests if they need to in the future.

CC: @darshanhs90

Reply:
I made some changes and the current version no longer mentions specific subscriptions or accounts.